base_tier_validation_correction/tests/test_tier_validation.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2018 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from datetime import timedelta
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests.common import SavepointCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidation(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TierTierValidation, cls).setUpClass()
+
+        cls.loader = FakeModelLoader(cls.env, cls.__module__)
+        cls.loader.backup_registry()
+        from odoo.addons.base_tier_validation.tests.tier_validation_tester import (
+            TierValidationTester,
+        )
+
+        cls.loader.update_registry((TierValidationTester,))
+        cls.test_model = cls.env[TierValidationTester._name]
+
+        cls.tester_model = cls.env["ir.model"].search(
+            [("model", "=", "tier.validation.tester")]
+        )
+
+        # Access record:
+        cls.env["ir.model.access"].create(
+            {
+                "name": "access.tester",
+                "model_id": cls.tester_model.id,
+                "perm_read": 1,
+                "perm_write": 1,
+                "perm_create": 1,
+                "perm_unlink": 1,
+            }
+        )
+
+        # Create users:
+        group_ids = [
+            cls.env.ref("base.group_system").id,
+            cls.env.ref("base_tier_validation_correction.group_tier_correction").id,
+        ]
+        cls.test_user_1 = cls.env["res.users"].create(
+            {"name": "John", "login": "test1", "groups_id": [(6, 0, group_ids)]}
+        )
+        cls.test_user_2 = cls.env["res.users"].create(
+            {"name": "Mike", "login": "test2"}
+        )
+        cls.test_user_3 = cls.env["res.users"].create(
+            {"name": "Mary", "login": "test3"}
+        )
+
+        # Create tier definitions:
+        cls.tier_def_obj = cls.env["tier.definition"]
+        cls.tier_def_obj.create(
+            {
+                "model_id": cls.tester_model.id,
+                "review_type": "individual",
+                "reviewer_id": cls.test_user_1.id,
+                "definition_domain": "[('test_field', '>', 1.0)]",
+            }
+        )
+
+        cls.test_record = cls.test_model.create({"test_field": 2.5})
```

Comment:
All this is in [the CommonTierValidation class in the base module](https://github.com/OCA/server-ux/blob/14.0/base_tier_validation/tests/common.py#L9), use that instead of copy-pasting. Psuedo-code:

```python
from odoo.addons.base_tier_validation.tests.common import CommonTierValidation


class TierValidationCorrectionTests(CommonTierValidation):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        # extra setup if needed...

    def test_01_()
        # ...
```